ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -110,7 +118,8 @@ interface AddressSuggestion {
       :host ::ng-deep .mat-radio-label {
           white-space: normal;
       }
-    `]
+    `],
+    changeDetection: ChangeDetectionStrategy.OnPush
 })
```

Comment:
First performance change: switched to using OnPush strategy from default strategy that uses zone.js. Don't have hard data to prove it, but looks a little faster. We have Observables for all the @Input() params so it should have worked. No code changes except adding this declaration in the two Mail Address components were needed to use OnPush.